tajo-storage/tajo-storage-hbase/src/main/java/org/apache/tajo/storage/hbase/HBaseStorageManager.java

```diff
@@ -312,6 +312,17 @@ public static Configuration getHBaseConfiguration(Configuration conf, TableMeta
           HBaseStorageConstants.META_ZK_QUORUM_KEY + "' attribute.");
     }
 
+    String zkPort = hbaseConf.get(HConstants.ZOOKEEPER_CLIENT_PORT);
```

Comment:
We already use the default value.
If users doesn't set zookeeper hosts or client port, Tajo uses properties of HBase configuration file. 
The above code just be used to check empty value.